frontend/src/app/layout/app-layout/app-layout.component.ts

```diff
@@ -1,6 +1,6 @@
 import {AfterViewInit, Component, HostBinding, OnInit, ViewChild} from '@angular/core';
 import {DefaultAuthService} from '../../core/services/auth/auth.service';
-import {NavigationEnd, NavigationStart, Router} from '@angular/router';
```

Comment:
Did you remove this on purose?

Reply:
It is not used

Reply:
Ok then, it just seemed a little off-topic

Reply:
Have you heard of `CTRL + ALT + L`? 😂

Reply:
Most likeyl Ctrl+Alt+O

Reply:
Yes, I have but the point of a review is to look at the code and look for everything that seems suspicious. This is the only way we can prevent bugs.

Reply:
Ctrl+Alt+L  does not remove imports, doesn't it?

Reply:
@needToRoll I agree but checking imports is the job of CI not code review.
@M4R1KU Yes it does if you enable optimize imports as part of it

Reply:
@M4R1KU @jmesserli you can configure the behaviour of the code formatter in your settings. You cand define wathever you want the imports to be optimized or not.